compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -18,7 +18,8 @@
 namespace mlir::iree_compiler::AMDAIE {
 
 static LogicalResult setRootConfigForPadPipeline(func::FuncOp entryPointFn,
-                                                 linalg::LinalgOp linalgOp) {
+                                                 linalg::LinalgOp linalgOp,
+                                                 AIEConfig cfg) {
```

Comment:
There were some pipelines (e.g padpack) that have this config as an argument (even though nothing is done with it) while others dont. I had to make this consistent so that they can use the same functions. I choose to go in the direction of making all of them use them with the reasoning that - yes currently the pipelines are not doing anything smart with the config, but if that changes, we have the plumbing for it.